samples/spring-eventsourced-counter/src/main/java/com/example/actions/Confirmed.java

```diff
@@ -0,0 +1,13 @@
+package com.example.actions;
+
+public class Confirmed {
+
+	private static Confirmed instance = null;
+
+	private Confirmed(){}
+	
+	public static Confirmed defaultInstance(){
+	  if(instance == null) instance = new Confirmed();
+	  return instance;	
+	}
+}
```

Comment:
yeah, it does look weird. I'm checking the source file and it's only one tab. What can this be? I'll find out.

Reply:
I was too confident saying I was going to find out. I did not :) I think it may be GitHub. I'll double-check after the merge to the main that it is one tab.